src/renderers/stateRenderer.js

```diff
@@ -157,6 +220,65 @@ function getRegions() {
     });
 }
 
+function getFacilities() {
+    
+    const FACILITY_TABLE_ID = 'state-facility'
+    const FUNCTIONAL = 'Functional'
+    const NON_FUNCTIONAL = 'Non Functional'
+
+    ipcRenderer.send("Facility:getFacilitiesByState", parseInt(getProcessArgObj()));
+    ipcRenderer.once("Facility:getFacilitiesByStateOK", (e, res) => {
+        $('#listOfFacilities').empty();
+        let table = `<table id="state-facility" class="table table-striped table-bordered table-sm" cellspacing="0" width="100%">
+        <thead>
+          <tr>
+            <th class="th-sm" id="state-facility-regionName">Region Name
+            </th>
+            <th class="th-sm" id="state-facility-facilityName">Facility
+            </th>
+            <th class="th-sm" id="state-facility-isFunctional" >Functional
+            </th>
+          </tr>
+        </thead>
+        <tbody id="StateFacilities">
+        </tbody>
+        </table>
+        `
+
+        $('#listOfFacilities').append(table);
+
+        if (Array.isArray(res) && res.length) {
+              res.forEach(facility => {
+                let facilityStatus = facility.isFunctional;
+                if(facilityStatus){
+                    facilityStatus = FUNCTIONAL
+                }else{
+                    facilityStatus = NON_FUNCTIONAL
+                }
+                let regionTemplate = `
+                    <tr>
+                        <td scope="col">${facility.regionName}</a></td>
+                        <td scope="col" >${facility.facilityName}</td>
+                        <td scope="col" >${facilityStatus}</td>
+                    </tr>`;
+
+                $('#StateFacilities').append(regionTemplate);
+            });
```

Comment:
Can we extract this into a function by itself?

Reply:
There's actually two in this case, one for the table and one that injects the data for the table

Reply:
The entire thing actually, the thing inside the latest `{}`, lines 252-266